commit d3fa5ea86e595cfccba09b7549f260a8b4d21cc7
Author: Mark Robert Miller <markrmiller@apache.org>
Date:   Sat Jan 28 00:24:32 2012 +0000

    try raising how long we will wait for balanced open/close
    
    git-svn-id: https://svn.apache.org/repos/asf/lucene/dev/trunk@1236939 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java b/solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java
index 82bf738..7afb4cf 100755
--- a/solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java
+++ b/solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java
@@ -132,7 +132,7 @@ public abstract class SolrTestCaseJ4 extends LuceneTestCase {
      // wait a bit in case any ending threads have anything to release
      int retries = 0;
      while (endNumOpens - numOpens != endNumCloses - numCloses) {
-       if (retries++ > 30) {
+       if (retries++ > 60) {
          break;
        }
        try {
